sdcm/utils/cloud_monitor/report.py

```diff
@@ -170,3 +170,9 @@ def _is_older_than_5days(create_time):
     @staticmethod
     def _is_older_than_7days(create_time):
         return create_time < pytz.utc.localize(datetime.utcnow() - timedelta(days=7))
+
+    def _is_user_be_skipped(self, instance):
+        return instance.owner == "qa" or instance.owner not in self.qa_users or instance.state != "running"
```

Comment:
`instance.state != "running"` is checked in `_is_instance_be_skipped`

Reply:
fixed